cypress/component/basic/enzyme/README.md

```diff
@@ -0,0 +1,98 @@
+# Enzyme examples
+
+This folder shows several examples from [Enzyme docs](https://enzymejs.github.io/enzyme/).
+
+In general if you are migrating from Enzyme to `cypress-react-unit-test`:
+
+- there is no shallow mounting, only the full mounting. Thus `cypress-react-unit-test` has `mount` which is similar to the Enzyme's `mount`
```

Comment:
Enzyme mount is rendering elements and keep React components information, so it is possible to query by react component name, which is unreliable. 

```suggestion
- there is no shallow mounting, only the full mounting. Thus `cypress-react-unit-test` has `mount` which is similar to the Enzyme's `render`. It renders the full HTML and CSS output of your component. 
```